feedback-examples/streamlit/main.py

```diff
@@ -76,11 +82,6 @@ def _get_openai_type(msg):
     # Re-hydrate memory on app rerun
     memory.chat_memory.add_message(msg)
```

Comment:
Is it intentional to do all this extra stuff instead of just using [StreamlitChatMessageHistory](https://python.langchain.com/docs/integrations/memory/streamlit_chat_message_history) which is persisted across re-runs and iterating through that? Wasn't sure if the expression chain / vanilla chain were supposed to stay UI agnostic.

I'm happy to add also swap that in this PR (or a follow-on) and simplify this app a bit if preferred!

Reply:
I probably should have just used that! Probably me being hasty